examples/pinterest.amp.html

```diff
@@ -4,6 +4,7 @@
   <meta charset="utf-8">
   <title>Pinterest examples</title>
   <link rel="canonical" href="amps.html" >
+  <meta name="pinterest" content="nopin" />
```

Comment:
I'm only looking at the example, not the code, but comparing it to the [amp-pinterest spec](https://github.com/ampproject/amphtml/blob/master/extensions/amp-pinterest/amp-pinterest.md), there is no mention of this meta tag. If it has specific meaning, it may be worth documenting there as well.